App.js

```diff
@@ -1,38 +1,27 @@
-import { StatusBar } from "expo-status-bar";
-import { StyleSheet, Text, View, ScrollView } from "react-native";
-import APICall from "./components/APICall";
-import Header from "./components/Header";
-
-export default function App() {
-  return (
-    <ScrollView>
-      <Header />
-      <APICall styleSheet={styles.reviews} />
-      <View style={styles.container}>
-        <Text>Hello World</Text>
-        <StatusBar style="auto" />
-      </View>
-    </ScrollView>
-  );
-}
+import { StyleSheet } from "react-native";
+import { ScrollView, View } from "react-native";
+// import CreateDeck from "./components/CreateDeck";
```

Comment:
Was this for the future create deck component ?

Reply:
We have the create deck component but we don't need to render it directly in app - it will likely be a child of either Nav or Decks